scimono-client/src/test/java/com/sap/scimono/client/UserRequestTest.java

```diff
@@ -75,6 +79,42 @@ public void setup() {
   public void closeResources() {
     wireMockServer.stop();
   }
+  
+  @Test
+  public void testReadUserWithoutOptionalUserName() throws IOException {
```

Comment:
Naming: Not OK
Suggestion: `testReadUserRequiringUserName`

Reply:
Done